src/internal/SetBuildNumber/Directory.Packages.props.pp

```diff
@@ -42,6 +42,7 @@
     <PackageVersion Include="System.Text.Encoding.CodePages" Version="4.6.0" />
 
     <PackageVersion Include="Microsoft.AspNetCore.Owin" Version="3.1.13" />
+    <PackageVersion Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" />
```

Comment:
Just remove this and the references to this package everywhere. They're only causing problems with no benefit.

Reply:
I really wanted to understand what was going on here... but I give up.

Reply:
I assume we're hitting another edge case in Nuget that no one else cares about.

Reply:
Yeah, CPVM is still "preview" so part of it is my own doing but I was hoping I could understand the root issue to possibly better handle something like it in the `wix.targets` in the future.

But you're probably right that it is a collection of edge cases no one really cares about.